app/models/project.js

```diff
@@ -117,7 +117,7 @@ export default class ProjectModel extends Model {
       milestone: milestone.dcpMilestone,
       displayName: milestone.displayName,
       dcpActualstartdate: milestone.dcpActualstartdate,
-      dcpActualenddate: milestone.dcpActualenddate,
+      dcpActualenddate: ((milestone.dcpActualenddate) ? milestone.dcpActualenddate : milestone.dcpPlannedcompletiondate),
```

Comment:
This seems like it should be template logic, not model logic. Setting the actual date to equal a planned date could cause trouble.

Reply:
I moved this logic to the template and controller so it's easier to see when the dates are actual vs planned.